src/corehost/cli/apphost/bundle/runner.cpp

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "runner.h"
+#include "extractor.h"
+#include "pal.h"
+#include "trace.h"
+#include "utils.h"
+
+using namespace bundle;
+
+manifest_t* runner_t::s_manifest = nullptr;
+int32_t runner_t::s_num_embedded_files = 0;
+pal::string_t runner_t::bundle_path;
+pal::string_t runner_t::s_bundle_id;
+pal::string_t runner_t::s_extraction_path;
+int8_t* runner_t::s_bundle_map = nullptr;
+size_t runner_t::s_bundle_length = 0;
```

Comment:
Can we instead make the runner instantiable and have a single global of type runner_t (or something similar)? Or maybe create a class just for the global state.
It would read nicer than what looks like a random collection of global variables.

Reply:
I thought about this option: making `runner_t` instantiable, and having a separate `bundle_info_t` holding global data. However, most of the above fields will end up being in transferred/copied to `bundle_info_t`.  Alternatively, we can have a global object of `runner_t` type.

Anyway, I'll change `runner_t` to be an instantiable class for now. 
We can create the global static data when it is actually used by a later change.